pkg/config/network.go

```diff
@@ -58,3 +58,11 @@ func getClusterDNS(serviceCIDR string) (string, error) {
 
 	return dnsClusterIP.String(), nil
 }
+
+func isValidIPAddress(ipAddress string) error {
```

Comment:
Being a validator function, it makes more since I think to simply return the bool value and generate the error at the caller's level.  Currently, the caller's already wrap this error with a more specific error anyway.

Reply:
Ah wait, I had another function here that returned an error, hence the `isValidIPAddress` return type, then switched to `net.ParseIP` which does not return an error but somehow kept the old signature. Now its updated.